pkgs/development/python-modules/ml-dtypes/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, numpy
+, pybind11
+}:
+
+buildPythonPackage rec {
+  pname = "ml-dtypes";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "jax-ml";
+    repo = "ml_dtypes";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-eqajWUwylIYsS8gzEaCZLLr+1+34LXWhfKBjuwsEhhI=";
+    fetchSubmodules = true;
+  };
+
+  propagatedBuildInputs = [
+    numpy
+    pybind11
```

Comment:
that is only used at build time and should go to nativeBuildInputs https://github.com/search?q=repo%3Ajax-ml%2Fml_dtypes%20pybind11&type=code

Reply:
Indeed ! Fixed.